packages/terra-clinical-result/docs/ClinicalResultBloodPressure.md

```diff
@@ -0,0 +1,95 @@
+# Terra Clinical Result Blood Pressure
+
+The Clinical Result is a starndardized view for single result values added to a Patient's Medical Chart. This is specifically for Blood Pressure Vital Sign results.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+```
+$ npm install terra-clinical-result
+```
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Component Features
+
+* [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
+* [Responsive Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#responsive-support)
+* [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
+* [LTR/RTL Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#ltr--rtl-support)
+ 
+<!-- Uncomment supported features.
+* [Internationalization Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#internationalization-i18n-support)
```

Comment:
Lets delete this if it's not supported

Reply:
Deleted https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13#diff-0748fd62687d573e95ea9081ee65da9eL21